pages/workspaces/new.tsx

```diff
@@ -47,7 +47,7 @@ const NewWorkspace = () => {
   });
 
   return (
-    <WorkspaceLayout>
+    <WorkspaceLayout workspaceId={null}>
```

Comment:
Since the new workspace page won't have a workspace ID, we set this to `null`. I didn't make it an optional field as it really is required. This is just a special case, hence the `null`.